distributions/nuxt-ts/src/index.js

```diff
@@ -0,0 +1,35 @@
+import { resolve } from 'path'
+import chalk from 'chalk'
+import consola from 'consola'
+import { existsSync, writeJSON } from 'fs-extra'
+import { register } from 'ts-node'
+import { prompt } from 'enquirer'
+
+export async function generateTsConfigIfMissing(rootDir) {
+  const tsConfigPath = resolve(rootDir, 'tsconfig.json')
+
+  if (!existsSync(tsConfigPath)) {
+    const { confirmGeneration } = await prompt({
+      type: 'confirm',
+      name: 'confirmGeneration',
+      message: `Missing ${chalk.bold.blue('tsconfig.json')} in ${rootDir === process.cwd() ? 'current directory' : chalk.bold.green(resolve(rootDir))}, generate it ?`,
+      initial: true,
+      skip: process.env.NODE_ENV === 'test'
+    })
+
+    if (confirmGeneration) {
+      await writeJSON(tsConfigPath, {
```

Comment:
Idea: We can use `.json5` for tsconfig and annotate it with some comments. Like links to docs or indicating it is an autogenerated file.

Reply:
(Not sure. But probably typescript supports json5 by default for `.json` files too)

Reply:
Gonna take a look, thanks

Reply:
@pi0 I think we can't do it, I tried things but got issues.
Gonna add a `consola.info` with a link to the extended `tsconfig.json`

Reply:
Ok. Thanks for your time on it ☺️